lib/cell.rb

```diff
@@ -0,0 +1,10 @@
+class Cell
```

Comment:
So, when I was thinking of a cell class it was more an instance of a cell has a coordinate and a mark, but this just moves literals to a different class.

Think about the grid data as being an array of cells, and each cell being made up of a coordinate and a mark.